libs/pages/services/src/lib/router/router.tsx

```diff
@@ -0,0 +1,14 @@
+import { Route, SERVICES_DEPLOYMENTS_URL, SERVICES_GENERAL_URL } from '@console/shared/router'
+import GeneralFeature from '../feature/general/general-feature'
+import DeploymentsFeature from '../feature/deployments/deployments-feature'
+
+export const ROUTER_SERVICES: Route[] = [
+  {
+    path: SERVICES_GENERAL_URL,
+    component: <GeneralFeature />,
```

Comment:
Here in the Subrouting of this page, we don't call any Pages, we call Feature components. Ok?